apps/component-examples/src/examples-server.ts

```diff
@@ -29,23 +69,7 @@ server.addRoute({
                 <h1>JustiFi Component Examples</h1>
                 <p>Enhanced component examples with live props editing and event logging</p>
               </div>
-              <div class="examples-grid">
-                <div class="example-card">
-                  <h3>Checkout Component</h3>
-                  <p>Complete checkout experience with payment method selection and billing information collection. Features live props editing and real-time event logging.</p>
-                  <a href="/checkout">View Example</a>
-                </div>
-                <div class="example-card">
-                  <h3>Business Details</h3>
-                  <p>Display business information with interactive editing capabilities and comprehensive event tracking.</p>
-                  <a href="/business-details">View Example</a>
-                </div>
-                <div class="example-card">
-                  <h3>Payment Details</h3>
-                  <p>Payment information display with real-time updates and detailed event logging.</p>
-                  <a href="/payment-details">View Example</a>
-                </div>
-              </div>
+              ${navigationContent}
```

Comment:
This looks great! I think that the old css classes can go away now, before we forget to remove them (`example-card` and `examples-grid`)